src/base.jl

```diff
@@ -35,7 +35,7 @@ Base.isvalid(s::ShortString, i::Integer) = isvalid(String(s), i)
 Base.iterate(s::ShortString) = iterate(String(s))
 Base.iterate(s::ShortString, i::Integer) = iterate(String(s), i)
 Base.lastindex(s::ShortString) = sizeof(s)
-Base.ncodeunits(s::ShortString) = ncodeunits(String(s))
+Base.ncodeunits(s::ShortString) = sizeof(s)
```

Comment:
Is this right?
If so:
I am not sure we should be overloading `sizeof` or just `ncodeunits`

Reply:
I stand corrected. `sizeof` and `ncodeunits` should be the same here. I was confused. I thought `ncodeunits` counted characters but `length` does